about/policy/sgid/index.md

```diff
@@ -25,26 +29,23 @@ When adding a new table to the SGID, a corresponding record will be added to the
 - A link the the data page on the AGRC website
 - A link to the SGID Open Data page
 
-The ArcGIS Online item will contain a curated summary, description, and tags while residing in its ISO category folder. If the data is authoritative, it will be marked as `authoritative`. If the data does not receive updates, it will be tagged as `static`. If the data has historical relevance **but is not the most recent data** it will be tagged as `shelved`.
-
-
-If the data is shared with the SGID Open Data, it will have the enable multiple formats download option selected to allow for File Geodatabases and other non default downloads.
+The ArcGIS Online Item will reside in the corresponding ISO category folder and contain the appropriate tags. If the data is authoritative, it will be marked as `authoritative`. If the data does not receive updates, it will be tagged as `static`. If the data has historical relevance **but is not the most recent data** it will be tagged as `shelved`. Note: the [AGOL Validator](https://github.com/agrc/agol-validator) project will transfer metadata from the Internal SGID database, including summary, description, and use limitations.
 
-The minimum metadata content will be populated.
+If the data is shared with the [SGID Open Data](https://opendata.gis.utah.gov/), it will have the enable multiple formats download option selected to allow for File Geodatabases and other non default downloads.
 
-- Purpose of data
-- Owner of the data
-- Contact person
-- Date loaded
-- Last update date
+The [minimum metadata content](https://docs.google.com/document/d/1VkXRwfSn6MraI1VeLfei5tg6je4bd2pp_Vh1JuST9xs/edit?usp=sharing) will be populated. When creating new metadata, it is recommended that you use the [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit).
 
 All empty geometries and duplicate records will be removed, with [Sweeper](https://github.com/agrc/sweeper) or some other form of automation.
 
+All new database domains will comply with the rule stating, domain values must match domain descriptions. See the [domain document](https://docs.google.com/document/d/12bdtmtv-ZVF9g-lFshbobx8DbKX9LfbL44HawLjFTVw/edit?usp=sharing) for more information on this effort.
+
 The ESRI default Resolution and Tolerance will be used.
 
 - XY Resolution: 0.0001 Meter
 - XY Tolerance: 0.001 Meter
 
+
+
```

Comment:
**nitpick**: let's remove the empty line

Reply:
good catch